plugins/inputs/teamspeak/README.md

```diff
@@ -0,0 +1,43 @@
+# Teamspeak 3 Input Plugin
+
+This plugin uses the Teamspeak 3 ServerQuery interface of the Teamspeak server to collect statistics of one or more
+virtual servers. If you are querying an external Teamspeak server, make sure to add the host which is running Telegraf
+to query_ip_whitelist.txt in the Teamspeak Server directory.
+
```

Comment:
I tried to find a good link with more info, but the best I could find is this pdf linked from go-ts3, do you know of any better reference material (preferably html)?  If not maybe we can link to the pdf, this will help if someone has additional questions about the meaning of the fields.

 http://media.teamspeak.com/ts3_literature/TeamSpeak%203%20Server%20Query%20Manual.pdf

Reply:
Yeah, the documentation directly from teamspeak for the query interface is quite bad.
There are txt files shipped with every server release containing plaintext documentation for each command (I wrote a small script generating a webpage from these [here](https://thor77.github.io/TeamspeakQueryDatabase/)).